extensions/parquet/base/src/main/java/io/deephaven/parquet/base/AbstractBulkValuesWriter.java

```diff
@@ -41,6 +41,18 @@ public int writeBulkVector(@NotNull final BUFFER_TYPE bulkValues,
         return applyDlAndRl(vectorSizes, rlEncoder, dlEncoder, nullsOffsets);
     }
 
+    /**
+     * This method is used to provide a specialized definition of NULL for PlainIntChunkedWriter. For other writers,
+     * this method does nothing
+     *
+     * @param nullDefinition The specialized value of NULL to consider while writing
+     */
+    @Override
+    public void setNull(int nullDefinition) {
+        // TODO Should I raise an error here since this should not be called, should only be called for the
```

Comment:
It's a pretty common pattern to throw an UnsupportedOperationException for functions that should never be called.